crates/widgets/src/window.rs

```diff
@@ -188,6 +188,7 @@ impl Template for Window {
                     .push_action(Action::FocusEvent(event));
                 true
             })
+            .on_activate(|_, _| println!("Activate"))
     }
 
```

Comment:
I think it's left for testing purposes?
Should be removed?

Reply:
Thanks. Yes.